arc-script/core/src/eval.rs

```diff
@@ -68,7 +68,11 @@ impl Script<'_> {
             dataflow: &mut dataflow,
             typer: info.typer.get_mut(),
         };
-        let _ = ast.body.eval(&mut ctx);
+        // TODO: Currently a hacky solution
+        // which just evaluates the last function of the script
```

Comment:
Why not do it as escript? Instead of just running the last script, look for a function called `main` which takes a list of strings as arguments (the command line parameters)

Reply:
It's a good point. Currently I have implemented it in a stupid way. After parsing the program assumes all symbol (strings) are resolved into identifiers (integers). It is hard to lookup the declaration of a symbol after parsing. I am thinking about how to redesign it all to support a module system with better support for name handling.